docs/section19/19-12.md

```diff
@@ -0,0 +1,47 @@
+**Schema Version:** 0.0.21  
+**Mandatory Rule:** True  
+**Rule ID:** 19-8         
+**Rule Description:** Demand control ventilation is modeled in the baseline design in systems with outdoor air capacity greater than 3000 cfm serving areas with an average occupant design capacity greater than 100 people per 1000 ft^2.     
+**Rule Assertion:** Options are Pass/Fail     
+**Appendix G Section:** Section G3.1.2.5 Excetion #1      
+**90.1 Section Reference:** None  
+
+**Data Lookup:** None  
+
+**Evaluation Context:** Each HeatingVentilationAirConditioningSystem Data Group  
+
+**Applicability Checks:** None  
+
+**Function Calls:**  
+1. get_hvac_zone_list_w_area()  
+
+
+## Rule Logic:  
+- Get dictionary with hvac systems and associated zones and areas: `hvac_zone_list_w_area_dict_b  = get_hvac_zone_list_w_area(B_RMR)`  
+- For each hvac_b system in B_RMR: `for hvac_b in B_RMR...Zone:`  
+    - Reset is_DCV_modeled to FALSE: `is_DCV_modeled = FALSE`  
+    - Get the hvac system minimum OA flow: `hvac_min_OA_flow = hvac_b.fan_system.minimum_outdoor_airflow`  
+    - Check whether DCV was modeled: `if hvac_b.fan_system.demand_control_ventilation_control != Null AND hvac_b.fan_system.demand_control_ventilation_control != "None": is_DCV_modeled_b = TRUE`  
+    - Check whether the outdoor air capacity (i.e., minimum OA CFM) is greater than 3,000 cfm, if yes then carry on: `if hvac_min_OA_flow > 3000:`  
+        - Get list of zones served: `zone_list_b = list(hvac_zone_list_w_area_dict_b[hvac_b.id]["Zone_List"].values())`  
+        - Get area served by the the HVAC system: `hvac_area_b = hvac_zone_list_w_area_dict_b[hvac_b.id]["TOTAL_AREA"]`  
+        - Reset total_hvac_sys_occupants_b to zero: `total_hvac_sys_occupants_b = 0`  
+        - For each zone associated with the HVAC system, get the number of occupants associated with the hvac system by looping through all the associated zones and spaces: `for zone_b in zone_list_b:`         
+            - For each space associated with the zone, get the number of occupants associated with the space: `for space_b in zone_b:`  
+                - Get number of occupants: `number_occ_b = space_b.number_of_occupants`  
+                - Find the maximum multiplier in the occupant_multiplier_schedule (schedule may need to be converted by RCT to do this): `max_multiplier_b = Max(space_b.occupant_multiplier_schedule.hourly_values)`  
+                - Calculate the max number of occupants: `max_number_occ_b = max_multiplier_b * number_occ_b`  
+                - Add to the total occupants associated with the HVAC system: `total_hvac_sys_occupants_b = total_hvac_sys_occupants_b + max_number_occ_b`  
+
+    - **Rule Assertion:** 
+    - Case 1: If hvac_min_OA_flow > 3000 AND total_hvac_sys_occupants_b/hvac_area_b > 100 AND is_DCV_modeled_b == TRUE then pass: `If hvac_min_OA_flow > 3000 AND total_hvac_sys_occupants_b/hvac_area_b > 100 AND is_DCV_modeled_b == TRUE: outcome = "PASS"`  
```

Comment:
it is 100 people / 1000 sqft, so it seems the `total_hvac_sys_occupants_b / hvac_area_b > 0.1` ?

Reply:
yes! good ctach